src/services/peopleSearch.ts

```diff
@@ -1,4 +1,6 @@
+import { useState } from 'react';
 import http from './http';
+import { forEach } from 'lodash';
```

Comment:
```suggestion
import { forEach } from 'lodash';
```
can you check if this import is being used? it shouldn't be as `typescript`/`javascript` has a inbuilt method for their array prototypes